squad-server/log-parser/player-connected.js

```diff
@@ -6,9 +6,22 @@ export default {
       time: args[1],
       chainID: args[2],
       playerSuffix: args[3],
-      steamID: logParser.eventStore['steamid-connected']
+      steamID: logParser.eventStore['steamid-connected'], // player connected
+      controller: logParser.eventStore['player-controller'] // playercontroller connected
     };
 
+    delete logParser.eventStore['steamid-connected'];
```

Comment:
We removed steamid-connected in this PR
I think this should point towards client-connected

Reply:
Planning on keeping this as is for now. Trying to play catch up with the PRs. If you want to add these changes in you are welcome to add your on PR for it.